crates/llama-cpp-bindings/build.rs

```diff
@@ -69,8 +72,27 @@ fn main() {
         println!("cargo:rustc-link-lib=hipblas");
     }
 
+    // By default, this value is automatically inferred from Rust’s compilation profile.
+    // For Windows platform, we always build llama.cpp in release mode.
+    // See https://github.com/TabbyML/tabby/pull/948 for more details.
+    #[cfg(target_os = "windows")]
+    config.profile("Release");
```

Comment:
this can be move inside of block 82 directly? without `#[cfg(...)]`

Reply:
Not exactly, `config.build()` in the next line will start the process of compiling. So we need to make the config ready before the build starts.

Alternatively, we could have following code:
```
    if cfg!(target_os = "windows") {
        config.profile("Release");
        let dst = config.build();
        println!(
            "cargo:rustc-link-search=native={}\\build\\{}",
            dst.display(),
            config.get_profile()
        );
    } else {
        let dst = config.build();
        println!("cargo:rustc-link-search=native={}/build", dst.display());
    }
```
Is this ok?